src/plugins/gui/scenes/EnterAmountScene.js

```diff
@@ -198,3 +215,28 @@ const getStyles = cacheStyles((theme: Theme) => ({
     resizeMode: 'contain'
   }
 }))
+
+// Check of string has no more than one comma or one period
+const checkAmount = (amount: string): boolean => {
+  // const dotSplit = amount.split('.')
```

Comment:
Lots of commented code here, any reason to keep around?

Reply:
Ah you caught me before I got a chance to rebase and remove comments. Check it out again